src/plugins/bookPlayer/plugin.js

```diff
@@ -271,40 +268,32 @@ export class BookPlayer {
                 const book = epubjs(downloadHref, {openAs: 'epub'});
                 const rendition = this.render('viewer', book);
 
-                this._currentSrc = downloadHref;
-                this._rendition = rendition;
-                const cancellationToken = {
-                    shouldCancel: false
-                };
-
-                this._cancellationToken = cancellationToken;
+                this.currentSrc = downloadHref;
+                this.rendition = rendition;
 
                 return rendition.display().then(() => {
                     const epubElem = document.querySelector('.epub-container');
                     epubElem.style.display = 'none';
 
                     this.bindEvents();
 
-                    return this._rendition.book.locations.generate(1024).then(async () => {
-                        if (cancellationToken.shouldCancel) {
-                            return reject();
-                        }
+                    return this.rendition.book.locations.generate(1024).then(async () => {
+                        if (this.cancellationToken) reject();
```

Comment:
I'm not a fan of brace-less if statements, but I don't know how common a pattern that is in our code-base.

Reply:
It's relatively common in this codebase. We can always add a rule later I suppose, but this repository is getting less active lately.